.github/ISSUE_TEMPLATE/scraper_bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Scraper bug report
+about: Report a scraper that is not working correctly
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Thanks for filing a bug report with us!  To help get the issue fixed, please fill in the information below.
```

Comment:
Maybe we should specify a website not being supported is not a bug, but should be instead handled with the other template.